packages/webapp/src/components/Task/TaskAnimalInventory/index.jsx

```diff
@@ -34,20 +34,14 @@ export default function PureTaskAnimalInventory({
 }) {
   const { t } = useTranslation();
   const ANIMAL_IDS = 'animalIds';
+  const preSelectedIds = persistedFormData.animalIds || history.location?.state?.animal_ids;
```

Comment:
I forget how history.location works so I thought there was a chance that it might stick around when the desired value is a clean slate [ ], but it seems to be working when navigating away then back so I think it was unfounded.